cetmix_tower_server/static/src/components/server_status/server_status_field.esm.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module */
+
+import {registry} from "@web/core/registry";
+import {StateSelectionField} from "@web/views/fields/state_selection/state_selection_field";
+
+import {STATUS_COLORS, STATUS_COLOR_PREFIX} from "../../utils/server_utils.esm";
```

Comment:
What do you create these constants for? You can set the values in the setup method without create constants.

Reply:
it's common practice for odoo

Why?

Constants make it clear that the values are configuration, not logic. The same values can be reused across multiple components or modules. You only need to update the value in one place instead of searching through your code. Logic stays in the component, while static data stays in a dedicated file (like utils.js). Prevents hardcoding the same values in multiple places.